controllers/state_of_the_world.go

```diff
@@ -63,15 +71,122 @@ func NewPolicyMachineryController(manager ctrlruntime.Manager, client *dynamic.D
 }
 
 func buildReconciler(client *dynamic.DynamicClient) controller.ReconcileFunc {
+	authorinoCrSubscription := &controller.Subscription{
+		ReconcileFunc: NewAuthorinoCrReconciler(client).Reconcile,
+		Events: []controller.ResourceEventMatcher{
+			{Kind: ptr.To(kuadrantv1beta1.KuadrantKind), EventType: ptr.To(controller.CreateEvent)},
+			{Kind: ptr.To(kuadrantv1beta1.AuthorinoKind), EventType: ptr.To(controller.DeleteEvent)},
+		},
+	}
 	reconciler := &controller.Workflow{
 		Precondition: NewEventLogger().Log,
 		Tasks: []controller.ReconcileFunc{
 			NewTopologyFileReconciler(client, operatorNamespace).Reconcile,
+			authorinoCrSubscription.Reconcile,
 		},
 	}
```

Comment:
This is fine for now. Probably what we'll want soon is:

```suggestion
	reconciler := &controller.Workflow{ // main
		Precondition: &controller.Workflow{ // logging, topology, and error handling
			Precondition: NewEventLogger().Log,
			Tasks: []controller.ReconcileFunc{
				NewTopologyFileReconciler(client, operatorNamespace).Reconcile,
			},
		},
		Tasks: []controller.ReconcileFunc{
			authorinoCrSubscription.Reconcile,
		},
	}
```

Reply:
This was something I was going to do after this was merged. I guess I could do it as part of this PR.